lightkurve/targetpixelfile.py

```diff
@@ -922,8 +923,14 @@ def get_model(self, star_priors=None, **kwargs):
         if 'prfmodel' not in kwargs:
             kwargs['prfmodel'] = self.get_prf_model()
         if 'background_prior' not in kwargs:
-            flux_prior = GaussianPrior(mean=np.nanmean(self.flux_bkg),
-                                       var=np.nanstd(self.flux_bkg)**2)
+            if np.all(np.isnan(self.flux_bkg)):  # If TargetPixelFile has no background flux data
+                # 5 sigma clip the flux until convergence, and use as an estimate for flux_bkg
+                clipped_flux = sigma_clip(self.flux, sigma=5, iters=None)
```

Comment:
If there are a lot of bright stars, sigma-clipping may end up rejecting the background pixels.  I think the more conventional method for background estimation is to use percentiles. How about this: 

````
if targetpixelfile has no background data:
    flux_prior = UniformPrior(lb=np.percentile(self.flux, 0), lb=np.percentile(self.flux, 50)
```